src/main.lua

```diff
@@ -72,11 +72,21 @@ function love.update(dt)
   player:update(ground, dt)
 
   -- detect if player is triggering new road creation and calculate location of new road
-  distance = love.physics.getDistance(player.fixture, road.frontier.fixture)
+  distance = love.physics.getDistance(player.fixture, road.frontier.main.fixture)
   -- collided with frontier
-  if distance < road.roadjog/10 then
+  while distance < 0.1 do
```

Comment:
I would create a variable called like `pavethreshold` or `jogthreshold` just to make it a bit more self documenting.